frontend/packages/otr/src/components/clerkInterpreter/overview/ClerkInterpreterDetailsFields.tsx

```diff
@@ -221,10 +223,25 @@ export const ClerkInterpreterDetailsFields = ({
       setDisplayFieldError({ ...displayFieldError, [field]: true });
     };
 
+  const isIndividualisedValue = (field: ClerkInterpreterTextFieldEnum) => {
+    const isIdentityNumber =
```

Comment:
Tästä voi tehdä hieman helppolukuisemman jos heti funktion alussa tehtäis identity check ja sille oma return
`if (field === ClerkInterpreterTextFieldEnum.IdentityNumber) {
  return true;
}`

Sitten jos yo. ei ole tosi niin palautetaan suoraan:

`return  isIndividualisedInterpreter &&
      [
        ClerkInterpreterTextFieldEnum.LastName,
        ClerkInterpreterTextFieldEnum.FirstName,
        ClerkInterpreterTextFieldEnum.NickName,
      ].includes(field);
`

Reply:
Fiksaan 👌